tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```diff
@@ -83,20 +83,39 @@ class ReplicationUtility {
     }
 
     deleteAllBlobs(containerName, keyPrefix, cb) {
-        const options = { include: 'metadata' };
-        this.azure.listBlobsSegmented(
-            containerName,
-            null,
-            options,
-            (err, result) => {
-                if (err) {
-                    return cb(err);
-                }
-                // Only delete the blobs put by the current test.
-                const filteredEntries = result.entries.filter(entry => entry.name.startsWith(keyPrefix));
-                return this._deleteBlobList(filteredEntries, containerName, cb);
-            },
-        );
+        const options = { includeMetadata: true };
+        // const iter = this.azure.getContainerClient(containerName)
+        //     .listBlobsFlat(options)
+        //     .byPage(/* { maxPageSize: 200 } */);
+        // const processPage = res => {
+        //     if (res.done) {
+        //         return cb(null);
+        //     }
+        //     const { blobItems } = res.value.segment;
+        //     const filteredEntries = blobItems.filter(blobItem => blobItem.name.startsWith(keyPrefix));
+        //     return this._deleteBlobList(filteredEntries, containerName, (err) => {
+        //         if (err) {
+        //             return cb(err);
+        //         }
+        //         return iter.next().then(processPage, cb);
+        //     });
+        // };
+        // iter.next().then(processPage, cb);
+
+        (async () => {
```

Comment:
Just wondering if here, it is really helpful to wrap the whole `deleteAllBlobs` logic inside a function... Why not changing this function to be async? As we return a callback anyway, it will work.